package.json

```diff
@@ -5,8 +5,7 @@
   "main": "lib/index.js",
   "types": "index.d.ts",
   "dependencies": {
-    "@types/express-jwt": "0.0.42",
```

Comment:
Can you double check you can move this to dev deps - it's referenced in you index.d.ts, so I'm not sure it can